cmd/main.go

```diff
@@ -51,8 +51,7 @@ import (
 	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
 	"github.com/kyma-project/lifecycle-manager/internal"
 	"github.com/kyma-project/lifecycle-manager/internal/controller"
-	kymametrics "github.com/kyma-project/lifecycle-manager/internal/controller/kyma/metrics"   //nolint:importas // ad-hoc import collision resolution
-	purgemetrics "github.com/kyma-project/lifecycle-manager/internal/controller/purge/metrics" //nolint:importas // ad-hoc import collision resolution
+	"github.com/kyma-project/lifecycle-manager/internal/pkg/metrics"
```

Comment:
Please the remove the `pkg` parent,  it's either `pkg` or `internal`. `metrics` is a nice enough package name for this

Reply:
what do you mean the `pkg` parent?

Reply:
What's the intent of creating a `pkg` subpackage within `internal`?

Reply:
As of clarification we can indicate shared internal package to put them in `internal/pkg` 👍